src/java.management/share/classes/com/sun/jmx/remote/internal/ServerNotifForwarder.java

```diff
@@ -346,7 +346,7 @@ public void terminate() {
 
     @SuppressWarnings("removal")
     private Subject getSubject() {
-        return Subject.getSubject(AccessController.getContext());
+        return Subject.current();
```

Comment:
Since `Subject::current` is not deprecated the annotation at line 347 above should be removed.